lib/integrations/CalDav/CalDavCalendarAdapter.ts

```diff
@@ -234,8 +222,8 @@ export class CalDavCalendar implements CalendarApiAdapter {
             ids.map(async (calId) => {
               return (await this.getEvents(calId, dateFrom, dateTo)).map((event) => {
                 return {
-                  start: event.startDate,
-                  end: event.endDate,
+                  start: event.startDate.toISOString(),
+                  end: event.endDate.toISOString(),
```

Comment:
Same as above

Reply:
@emrysal is the availability expert now 😊